### ISSUE TITLE
wrong key mappings for pop up menu,  keyboard focus is not set correctly

### ISSUE THREAD

**Issue by muphil** [FelisCatus/SwitchyOmega#1049]
I found two new issues in using this extension recently.

For the first one, please take a look at here: 
https://github.com/FelisCatus/SwitchyOmega/blob/master/omega-web/src/popup/js/keyboard.js#L9-L10
I guess it was a typo, so maybe these two key mappings should be exchanged?

Another issue is that when the pop up menu is triggered, the current profile does not have the keyboard focus. It's very inconvenient if you switch profiles by shortcuts.

**Comment by FelisCatus** [comment]
Whoops. It is definitely a typo. How could I make such a mistake when I myself is a vim user? :wink: 

As for the focus, yes, I will implement that soon.

**Comment by FelisCatus** [comment]
Fixed in 2.4.6.
